package.json

```diff
@@ -42,8 +42,11 @@
         "@react-native-community/clipboard": "^1.5.1",
         "@react-native-community/datetimepicker": "^3.5.0",
         "@react-native-community/masked-view": "^0.1.11",
+        "@react-native-community/netinfo": "^6.0.0",
         "@react-native-picker/picker": "^1.16.1",
+        "@sentry/react-native": "^2.6.1",
         "fast-text-encoding": "^1.0.3",
+        "pusher-js": "^7.0.3",

```

Comment:
Why must the storybook depend on pusher, onesignal or netinfo?

Reply:
Because it imports commons-native. If we don't include this, it sends out errors

Reply:
Errors or warnings?

Reply:
Errors

Reply:
please try checking if they can be made optional

Reply:
Update: I checked by using optionalDependencies, it ends up with the same problem. The only way I found to make this work was to add a try/catch in commons-native when importing the dependencies (sentry, onesignal and pusher), which would catch the error if they are not defined. Something like: 
```
let Pusher;
try {
    Pusher = require("pusher-js/react-native");
} catch(e) {}
```

...continuing the investigation